packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -737,9 +797,13 @@ export class SDKClient {
       );
 
       if (!transactionResponse) {
-        throw predefined.INTERNAL_ERROR(
-          `${requestDetails.formattedRequestId} Transaction execution returns a null value: transactionId=${transaction.transactionId}, callerName=${callerName}, txConstructorName=${txConstructorName}`,
-        );
+        if (sdkClientError.isConnectionDropped() || sdkClientError.isTimeoutExceeded()) {
```

Comment:
Q: do we need to check on the type of `sdkClientError`?
Is the point to just return an `sdkClientError` if applicable and a general error otherwise.
If so you only need to check for a valid `sdkClientError` object

Reply:
My idea here was to check the message of the error via these methods. So we want to throw an SDKClientError and catch this particular error in the upper level method, and we want to do this only for a timeout exceeded or connection dropped, thats when we observed a trnasaciton being successful but the SDK client returns an error

Reply:
Okay